tt_metal/impl/dispatch/memcpy.hpp

```diff
@@ -15,6 +15,27 @@
 #include <emmintrin.h>
 #endif
 
+#define LOAD_STREAM_32                                                                 \
```

Comment:
Nit: Would prefer `#define LOAD_STREAM_32()` since `LOAD_STREAM_32();` as a statement reads more intuitively than `LOAD_STREAM_32;` Also run less risk of unintended symbol name collision causing erroneous macro expansions in other places (though I do note the `#undef` at the end of the file).